subsys/bluetooth/controller/ticker/ticker.c

```diff
@@ -1983,6 +1964,63 @@ u32_t ticker_job_idle_get(u8_t instance_index, u8_t user_id,
 	return user_op->status;
 }
 
+/**
+ * @brief Set ticker node priority
+ *
+ * @param instance_index     Index of ticker instance
+ * @param user_id	     Ticker user id. Used for indexing user operations
+ *			     and mapping to mayfly caller id
+ * @param ticker_id	     Id of ticker node to set priority on
+ * @param priority	     Priority to set. Range [-128..127], default is 0.
+ *			     Lover value equals higher priority. Setting
+ *			     priority to -128 (TICKER_PRIORITY_CRITICAL) makes
+ *			     the node win all collision challenges. Only one
+ *			     node can have this priority assigned.
```

Comment:
What happens if >=2 nodes have critical priority?

Reply:
In that case it's first-come first-served. The critical priority is not guaranteed honored.